data/json/snippets/children_notes.json

```diff
@@ -130,11 +130,11 @@
       },
       {
         "id": "note_child_47",
-        "text": "\"I'm alone in this shelter now…  i can see some people outside but they are just standing there really quiet and not doing anything.  The monsters seems to ignore them too, maybe they will help me?\""
+        "text": "\"I'm alone in this shelter now… i can see some people outside but they are just standing there really quiet and not doing anything.  The monsters seems to ignore them too, maybe they will help me?\""
       },
       {
         "id": "note_child_48",
-        "text": "\"In case of emergency fill out this form so that you may present it to emergency personnel for identification purposes.  Keep this form with your emergency supplies.  If You need to request new copies of this form, speak with the school safety office.  Remember to keep calm, and stay in one location if possible, this will help emergency personnel to find you.  If this form is found please contact Oak Grove Elementary School.  Name: Aiden.  Parent or Guardian: Mommy Daddy.  Phone number where you can be reached: I don't remember I think it has a 5 in it.  Teacher: Miss daisy.  Current location: I dont no I'm scared, Im hear all alone.  Medical Conditions i.e diabetes: I dont no what that is.  Additional Information: Why is everyone acting so weird?!\""
+        "text": "\"In case of emergency fill out this form so that you may present it to emergency personnel for identification purposes.  Keep this form with your emergency supplies. If You need to request new copies of this form, speak with the school safety office.  Remember to keep calm, and stay in one location if possible, this will help emergency personnel to find you.  If this form is found please contact Oak Grove Elementary School.  Name: Aiden.  Parent or Guardian: Mommy Daddy.  Phone number where you can be reached: I don't remember I think it has a 5 in it.  Teacher: Miss daisy.  Current location: I dont no I'm scared, Im hear all alone.  Medical Conditions i.e diabetes: I dont no what that is.  Additional Information: Why is everyone acting so weird?!\""
```

Comment:
```suggestion
        "text": "\"In case of emergency fill out this form so that you may present it to emergency personnel for identification purposes.  Keep this form with your emergency supplies.  If You need to request new copies of this form, speak with the school safety office.  Remember to keep calm, and stay in one location if possible, this will help emergency personnel to find you.  If this form is found please contact Oak Grove Elementary School.  Name: Aiden.  Parent or Guardian: Mommy Daddy.  Phone number where you can be reached: I don't remember I think it has a 5 in it.  Teacher: Miss daisy.  Current location: I dont no I'm scared, Im hear all alone.  Medical Conditions ie diabetes: I dont no what that is.  Additional Information: Why is everyone acting so weird?!\""
```